docs(changelog): changelog 2022-12-29 (#1145)
scaleway/docs-content: changelog/observability/december2022/2022-12-22-added-object-storage.mdx

---
title: The Object Storage managed Dashboard is now available!
status: fixed
author:
    fullname: 'Join the #observability-beta channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-12-15
category: observability
product: cockpit
---

You can now monitor all your [Object Storage](https://www.scaleway.com/en/object-storage/) buckets in Observability.

> The managed dashboard give you overview on the following metrics:

Reviewer: ```suggestion
The managed dashboard gives you an overview on the following metrics:
```

- Total number of objects
- Total numbers of parts
- Total bucket usage
- Bucket usage per storage class
- Total ingress internal
- Total egress internal

> - Total bandwith internal

Author: ```suggestion
- Total bandwidth internal
```

- Total ingress external
- Total egress external

> - Total bandwith per type

Author: ```suggestion
- Total bandwidth per type
```
